packages/core/test/insured-bridge/e2e/InsuredBridge.e2e.js

```diff
@@ -80,10 +85,16 @@ describe("Insured bridge e2e tests", () => {
     const l1StandardBridgeAddress = await l2StandardBridge.l1TokenBridge();
     l1StandardBridge = factory__L1StandardBridge.connect(l1Wallet).attach(l1StandardBridgeAddress);
   });
+  beforeEach(async () => {
```

Comment:
I really hate that I needed this but it's too flaky otherwise.

Reply:
I'm going to spend time tomorrow working on removing this. It cant be flaky in CI

Reply:
If you do end up keeping it why not just move it into the existing `beforeEach` block